setup.py

```diff
@@ -15,6 +15,11 @@
 except ImportError:
   from distutils.core import setup, Extension
 
+if sys.version_info[0] == 2 and sys.version_info[1] < 7:
+    raise Warning('This software is not tested on your Python.  Be careful!')
+if sys.version_info[0] >= 3 and sys.version_info[1] < 3:
+    raise RuntimeError('The C API from Python 3.3+ is required.')
```

Comment:
This would fail on Python 4.0; it should be `if sys.version_info[0] >= 3 and sys.version_info < (3, 3):`